tmt/utils/__init__.py

```diff
@@ -6661,3 +6664,99 @@ def render_rst(text: str, logger: Logger) -> str:
     document.walkabout(visitor)
 
     return visitor.rendered
+
+
+def import_jira() -> None:
```

Comment:
Please, use `ModuleImporter`, see https://github.com/teemtee/tmt/blob/e93d03bbc2424fa0d5bb337753dca9568b53a016/tmt/steps/report/junit.py#L34 and how it's used in the code, https://github.com/teemtee/tmt/blob/e93d03bbc2424fa0d5bb337753dca9568b53a016/tmt/steps/report/junit.py#L65